examples/synthea-breast-cancer/Ali_Laiba_syntheaEDA/SyntheaEDA.ipynb

```diff
@@ -0,0 +1,996 @@
+      "text/plain": [
+       "<Figure size 432x288 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "sns.heatmap(corr)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can see that most of the fields don't seem to be correlated with the others, however, the *stage* category does seem to be slightly different, hence why we will try to use this header as the output for our classifier"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "___\n",
+    "In the end, we find that a suitable list of inputs for our classifiers can be:\n",
+    "- surgical: Number of surgeries performed on the patient\n",
+    "    - int between \\[0 .. \\*\\]\n",
+    "- radiation: Number of radiation therapy sessions performed on the patient\n",
+    "    - int between \\[0 .. \\*\\]\n",
+    "- cancerStatus: Whether or not the patient's condition improved\n",
+    "    - int between \\[0 .. 1\\]\n",
+    "- diagnosisAge: At what age the person was diagnosed with cancer\n",
+    "    - float between \\[0.0 .. \\*\\]\n",
+    "- primary: The patient's primary tumour category\n",
+    "    - int between \\[0 .. 4\\]\n",
+    "- nodes: The patient's regional node category\n",
+    "    - int between \\[0 .. 3\\]\n",
+    "- numberOfMeds: The number of medications given to the patient\n",
+    "    - int between \\[0 .. \\*\\]\n",
+    "\n",
+    "The output we determined to be optimal for our analysis was:\n",
+    "- stage: The clinical cancer stage of the patient\n",
+    "    - int between \\[0 .. 4\\]\n",
+    "\n",
+    "Finally, we now decide on what classifier to use on our data. We took a look at several different classifiers, from Support Vector Classifiers (SVC) to Naive Bayesian Classifiers. We ended up choosing our default classifier to be the Decision Tree Classifier, although we were quite interested in the Random Forest, KNN, Support Vector and Naive-Bayesian classifiers as alternative options. We chose to do this because Decision Trees are [easy to interpret, require minimal processing and are robust](https://towardsdatascience.com/decision-tree-classifier-explained-in-real-life-picking-a-vacation-destination-6226b2b60575#:~:text=Even%20though%20Decision%20trees%20is,all%20types%20of%20data%20nicely.)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**We now download the collected data into a useable file format for further classification**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df.to_csv(path_or_buf='SyntheaEDA.csv', index=False)"
```

Comment:
I like that you saved the dataframe built in the EDA and reused it in the training notebook, very helpful.
In the future you might want to try out `pickling` instead of a csv to accomplish this, I don't know a ton about it but it's the more proper-data-science-y way of sharing data between notebooks. Here's an article on it: https://betterprogramming.pub/dont-fear-the-pickle-using-pickle-dump-and-pickle-load-5212f23dbbce